### PR TITLE
Fix contact ID

### PR DESCRIPTION
**Pull request by hakujitsu**
This PR aims to fix the implementation of a contact index.

- Current contact ID is now a variable that will be initialised upon startup
- When AddCommand is executed, it will get the current contact ID from Model, and calculate the next contact id by incrementing it by 1.
- The AddPerson method in ModelManager will increment the current contact ID after successful addition of the person to the list.

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -27,6 +27,8 @@
  * Represents the in-memory model of the address book data.
  */
 public class ModelManager implements Model {
+    private int currentContactId = 0;
```
Comment (AaronnSeah) [inline src/main/java/seedu/address/model/ModelManager.java]: You could consider renaming to latestContactId? I think the name can be clearer about the intention of this variable.

**Review by AaronnSeah — COMMENTED** [review]
LGTM, will leave approving review to Jin ming.

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (hakujitsu) [reply src/main/java/seedu/address/model/ModelManager.java]: Renamed as suggested!

**Review by jmleong666 — APPROVED** [review]
LGTM!

**Pull request merged**